fly.toml.sample

```diff
@@ -0,0 +1,46 @@
+kill_signal = "SIGTERM"
+kill_timeout = 5
+processes = []
+
+[env]
+  INTEGRATED_TCP_TURN_PORT = "49999"
```

Comment:
I believe this port needs to be in `services.ports`, doesn't it?

Reply:
You're right, it should be. However this variable isn't used anywhere (other than parsing it), so I am not sure if we should set it at all.